plugins/node/opentelemetry-instrumentation-express/src/instrumentation.ts

```diff
@@ -244,6 +245,23 @@ export class ExpressInstrumentation extends InstrumentationBase<
         const span = instrumentation.tracer.startSpan(spanName, {
           attributes: Object.assign(attributes, metadata.attributes),
         });
+
+        if (
+          instrumentation.getConfig().requestHook &&
+          metadata.attributes[AttributeNames.EXPRESS_TYPE] ===
```

Comment:
i think we can generalize the hook to all layer type WDYT ? we just need to be sure we export the `ExpressLayerType` so that end user can use the correct name